carousel.css

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
The ".List" stylings should go in their own "List.css" file. Anything Carousel specific should also be labeled as such. So if the arrows are only meant to be apart of the Carousel, and not used elsewhere, They should be elements of the Carousel ("Carousel__arrow", "Carousel__arrow--left, "Carousel__arrow--right).